packages/itwin/property-grid/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@itwin/property-grid-react",
-  "version": "0.11.0-dev.0",
```

Comment:
I don't think you need to remove this because it might skip this minor and phbkish as 0.12. But we want to publish as 0.11

Reply:
this comment applied to every single pkg. i will fix